config/vufind/EDS.ini

```diff
@@ -335,6 +335,15 @@ AU = None
 ; This repeatable setting can be used to filter out values using group codes:
 ;excludeGroup[] = 'URL'
 
+; This section can be used to order items based on their labels.
+[ItemCoreOrder]
```

Comment:
I was not sure what the word "Core" meant here, but I see another section of EDS.ini says 

> ; Similar to [ItemGlobalFilter] but applied only for record view.
> [ItemCoreFilter]

So I think the word is misleading then, since the ItemCoreOrder as written applies to both the results page and the record view.  Would ItemGlobalOrder be correct?

Reply:
I've pushed this change to ItemGlobalOrder in my follow-up patchset, and switched EDS.php to use that new config name and have corrected variable names.